### DISCUSSION TITLE
README info note about F3 (reset) vs. exit and restart

### DISCUSSION THREAD

**Discussion by PL1-Arcade** [Jakobud/skipstartupframes#9] [General]
**NOTE:**  On some games, the number of frames to reach a specific point in the boot sequence is lower for an F3 (reset) compared to exiting the game and restarting it.  For best results, please verify any Frame Target changes with a restart.

One game where this is very obvious is Dangerous Seed. (dangseed,990)
\- When you first launch the game or exit and restart, Frame Target 990 is at the screen where the Namco letters start to fly in.
\- When you F3 (reset) the game, Frame Target 990 is at the next screen with the Dangerous Seed logo -- about 9 seconds later.

Scott

**Comment by Jakobud** [comment]
I have noticed this as well but I don't think there is any way around it.

MAME's LUA API cannot differentiate between a start and a reset so there is no way for the plugin to know how many frames to skip.

I asked the MAMEDev's about this a couple weeks ago:

https://github.com/orgs/mamedev/discussions/161#discussioncomment-11339377

There is no official solution at the moment. There is however one workaround I have thought about that I may get to work.

**Comment by PL1-Arcade** [comment]
Yeah, I just figured that it might be something to mention since you're working on the README.

The good news is that holub's suggestion of `Shift`+`F3` hard reset seems to work for this and (mostly) the slow motion issue #8.


Scott

**Comment by Jakobud** [comment]
I figured out a solution to this. Big release coming soon.
